internal/pkg/template/templates/workloads/partials/cf/http-listener.yml

```diff
@@ -33,7 +33,7 @@ HTTPRulePriorityAction:
 {{- range $i, $rule := .ALBListener.Rules}}
 HTTPListenerRule{{ if ne $i 0 }}{{ $i }}{{ end }}:
   Metadata:
-    'aws:copilot:description': 'A HTTP listener rule for forwarding HTTP traffic'
+    'aws:copilot:description': 'A HTTP listener rule for path {{$rule.Path}} for forwarding HTTP traffic'
```

Comment:
nit: maybe ⬇️  Do our fellow english experts have opinion on this 😂 ? @huanjani  @dannyrandall 
```suggestion
    'aws:copilot:description': 'An HTTP listener rule for forwarding HTTP traffic on path {{$rule.Path}} '
```

Reply:
Me offended to not included in dis list

Reply:
🥺 🥺 🥺

Reply:
It think it's "for path ..." but also would like english experts to weigh in.

Reply:
I'm going to throw another suggestion in here, with the additional grammar note that it would be good to surround the path string with quotes or backticks
```suggestion
    'aws:copilot:description': 'A listener rule for handling HTTP traffic for path `{{$rule.Path}}`'
```

Reply:
I like your option @bvtujo, but starting with `An HTTP listener rule....` is more clear as per my opinion.

Reply:
Am I too late to the party?
"An HTTP listener rule for handling traffic for path `{{$rule.Path}}`" is my vote.

Reply:
The compromise candidate 🤝